nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/feature/ColdStorageIntegrationFeature.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (C) Copyright 2021 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Abdoul BA<aba@nue.com>
+ */
+
+package org.nuxeo.coldstorage.feature;
+
+import org.nuxeo.coldstorage.ColdStorageFeature;
+import org.nuxeo.coldstorage.auth.ColdStorageS3TestHelper;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+/**
+ * Feature that allows integration tests: Using AWS S3 or LocalStack
+ * {@link  ColdStorageS3TestHelper}
+ *
+ * @since 2.0
+ */
+public class ColdStorageIntegrationFeature extends ColdStorageFeature {
+    @Override
+    public void afterTeardown(FeaturesRunner runner) throws Exception {
+
+    }
+
+}
```

Comment:
Do we still need this integration feature. We can simplify our life for now and rely only to S3 integration test